test/recipes/95-test_external_pkcs11_provider_data/pkcs11-provider.sh

```diff
@@ -63,6 +63,8 @@ echo "------------------------------------------------------------------"
 echo "Running tests"
 echo "------------------------------------------------------------------"
 
+# We run test on Fedora with previous version of Kryoptic, so disable ML-DSA test
+export SUPPORT_ML_DSA=0
```

Comment:
Why do we have this change in this PR? It seems unrelated.

Reply:
No, we can't do it as of now, it should be changed on changing the Fedora version